syntax = "proto3";

package NProto;

message TPutRequest {
    uint64 request_id = 1;
    uint64 key = 2;
    string offset = 3;
}

message TPutResponse {
    uint64 request_id = 1;
}

message TGetRequest {
    uint64 request_id = 1;
    uint64 key = 2;
}

message TGetResponse {
    uint64 request_id = 1;
    string offset = 2;
}
